common/test/acceptance/pages/studio/settings_graders.py

```diff
@@ -14,3 +14,91 @@ class GradingPage(CoursePage):
 
     def is_browser_on_page(self):
         return self.q(css='body.grading').present
+
+    def letter_grade(self, selector):
+        """
+        Returns: first letter of grade range on grading page
+        Example: if there are no manually added grades it would
+        return Pass, if a grade is added it will return 'A'
+        """
+        return self.q(css=selector)[0].text
+
+    def click_new_grade_button(self):
+        """
+        Clicks new grade button
+        """
+        self.q(css='.new-grade-button').click()
+        self.save_changes()
+
+    def click_remove_grade(self):
+        """
+        Clicks remove grade button
+        """
+        # Button displays after hovering on it
+        btn_css = '.remove-button'
+        self.browser.execute_script("$('{}').focus().click()".format(btn_css))
+        self.wait_for_ajax()
+        self.save_changes()
+
+    def remove_all_grades(self):
+        """
+        Removes all grades
+        """
+        while len(self.q(css='.remove-button')) > 0:
+            self.click_remove_grade()
```

Comment:
Do you need a wait statement? Is there any other action on that page that you could wait for?